styles.css

```diff
@@ -1,3 +1,13 @@
+*{
```

Comment:
Please indent your code

```suggestion
* {
```